pkg/cvo/cvo_test.go

```diff
@@ -3245,3 +3251,189 @@ func fakeClientsetWithUpdates(obj *configv1.ClusterVersion) *fake.Clientset {
 	})
 	return client
 }
+
+func Test_loadReleaseVerifierFromConfigMap(t *testing.T) {
+	redhatData, err := ioutil.ReadFile(filepath.Join("..", "verify", "testdata", "keyrings", "redhat.txt"))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tests := []struct {
+		name          string
+		update        *payload.Update
+		want          bool
+		wantErr       bool
```

Comment:
Neither `want` nor `wantErr` are very specific about what came back.  Can we make `wantErr` a regexp or string matching the expected error message?  And `want` something about the set of loaded keys?